pyproject.toml

```diff
@@ -117,7 +117,7 @@ xlwt = "1.2.0"
 "zope.sqlalchemy" = "^1.2"
 
 [tool.poetry.dev-dependencies]
-coverage = "4.0.3"
+coverage = "5.1"
```

Comment:
On CGAP the mark was >=5.1, is there a reason for the discrepancy here?

Reply:
Nope. Careless on my part. I'll fix that.

Reply:
In fact that will get coverage 5.2, but that's good enough and is what CGAP is proposed to do. It's up testing now. I'll merge it when it goes green.